gapic/samplegen/samplegen.py

```diff
@@ -214,7 +242,7 @@ def validate_and_transform_request(
             field_assignment_copy = dict(field_assignment)
             input_param = field_assignment_copy.get("input_parameter")
             if input_param:
-                self._handle_lvalue(input_param)
+                self._handle_lvalue(input_param, str)
```

Comment:
Not quite following this. `input_parameter` isn't always a `string`, it can be any primitive types IIUC.

Reply:
Yes, but from the point of view of the `validate_expression` it doesn't matter: all that the code needs to know is that it's _not_ a `MessageType`. Input parameters can be enums though, which I plan on addressing in a subsequent review.

Reply:
Okay that makes sense. Would you mind adding a comment here?

Reply:
Sure thing.